fem/pgridfunc.hpp

```diff
@@ -409,6 +409,12 @@ class ParGridFunction : public GridFunction
        the local dofs. */
    virtual void Save(std::ostream &out) const;
 
+   /// Convenience function to save the GridFunction and assoicated Mesh to
+   /// files. The files will be given suffixes according to the MPI rank.
+   /// The given @a precision will be used for ASCII output.
+   virtual void SaveWithMesh(const char *sol_f, const char *mesh_f,
+                             int precision=8) const;
+
```

Comment:
Could you add an option to save it as one file (one for the mesh and one for the gf)?

Reply:
I added `ParGridFunction::SaveAsOneWithMesh` to do this (I think it makes more sense as a separate function rather than as a flag passed to `ParGridFunction::SaveWithMesh`, since it overrides `GridFunction::SaveWithMesh`, which doesn't make sense with a "save as one" option).